web/package.json

```diff
@@ -56,7 +56,9 @@
     "parserOptions": {
       "ecmaVersion": 2020
     },
-    "rules": {},
+    "rules": {
+      "no-console": "off"
```

Comment:
I recommend still having a warning so console.log doesn't go unnoticed:

`    'no-console': ["warn", {
      allow: ["warn", "error"]
    }],`

Reply:
Changed in [d3b8233](https://github.com/atlascope/atlascope/pull/136/commits/d3b82334e132ed913323c5afa983c4c00fd8d4f8)